designs/tripleo/osc-tripleo.md

```diff
@@ -176,6 +183,30 @@ path/directory on overcloud node and the TLS certificate is copied to this locat
 
 >>c. Do we need to hard code the target path for storing the certificate? If so how is this used by puppet?
 
+#### OSC Network Configuration
+1. Add an entry in the file `network/service_net_map.j2.yaml` under the `ServiceNetMapDefaults` section for `OSCNetwork` like below. This is required to prevent OSC service to be part of provisioning network.
+    ```yaml
+        ServiceNetMapDefaults:
+            default:
+                ...
+                OSCNetwork: internal_api
```

Comment:
why OSC need to be part of internal_api network? what is the use-case in OSC that calls for internal network?

Reply:
For example, while adding VC we are adding keystone authentication details, then it needs to connect to keystone service for authorization.
Among all the networks available in tripleO ovecloud internal_api is looking most suitable one for OSC. But still needs to be confirmed by deploying OSC in TripleO setup. That's why I mentioned this in "Assumption" part.